src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByAuthorizationCodeParameterBuilder.cs

```diff
@@ -101,5 +101,16 @@ public AcquireTokenByAuthorizationCodeParameterBuilder WithPkceCodeVerifier(stri
             Parameters.PkceCodeVerifier = pkceCodeVerifier;
             return this;
         }
+
+        /// <summary>
+        /// 
```

Comment:
Need to add info

Reply:
@jmprieur - thoughts on adding this method on all AcquireToken* methods?

Reply:
@trwalke - I suggest we add this to all AcquireToken* methods, just like we do for `WithExtraQueryParameters`. Also, I would add it to the namespace `microsoft.identity.client.advanced`, to not polute the existing API, which is already too complicated.